js/client/modules/@arangodb/inspector.js

```diff
@@ -594,33 +595,116 @@ function getServerData(arango) {
             agencyState = arango.GET('_api/agency/state');
           }
           const status = arango.GET('_admin/status');
+          const engine = arango.GET('_api/engine/stats');
           const time = require('internal').time();
 
-          var tmp = executeExternalAndWait(
-            '/bin/bash', ['-c', 'date -u "+%Y-%m-%d %H:%M:%S %Z" | tee /tmp/inspector-date.out > /dev/null']);
-          const date = fs.readFileSync('/tmp/inspector-date.out', 'utf8').slice(0,-1);
-          tmp = executeExternalAndWait(
-            '/bin/bash', ['-c', 'dmesg | tee /tmp/inspector-dmesg.out > /dev/null']);
-          const dmesg = fs.readFileSync('/tmp/inspector-dmesg.out', 'utf8').slice(0,-1);
-          tmp = executeExternalAndWait(
-            '/bin/bash', ['-c', 'df -h | tee /tmp/inspector-df.out > /dev/null']);
-          const df = fs.readFileSync('/tmp/inspector-df.out', 'utf8').slice(0,-1);
-          tmp = executeExternalAndWait(
-            '/bin/bash', ['-c', 'cat /proc/meminfo | tee /tmp/inspector-meminfo.out > /dev/null']);
-          const meminfo = fs.readFileSync('/tmp/inspector-meminfo.out', 'utf8').slice(0,-1);
-          tmp = executeExternalAndWait(
-            '/bin/bash', ['-c', 'uptime | tee /tmp/inspector-uptime.out > /dev/null']);
-          const uptime = fs.readFileSync('/tmp/inspector-uptime.out', 'utf8').slice(0,-1);
+          var uname, date, dmesg, df, meminfo, cpuinfo, uptime, release, top, du,
+              docker = false; 
+
+          // Operating system calls
+
+          const prefix =
```

Comment:
We normally use `fs.join(...)` for building filenames. I would prefer this here too.